site/en/docs/lighthouse/seo/canonical/index.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: 'layouts/doc-post.njk'
+title: Document does not have a valid `rel=canonical`
```

Comment:
Should this have backticks? They're not automatically converted in the staged site.

Reply:
web.dev currently converts these in titles and such, but I guess d.c.c. doesn't. I'll remove them.